[2025-10-20T16:48:21.982+0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T16:48:24.477+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_etl.transform_GOOGL manual__2025-10-20T13:45:40+00:00 [queued]>
[2025-10-20T16:48:25.588+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_etl.transform_GOOGL manual__2025-10-20T13:45:40+00:00 [queued]>
[2025-10-20T16:48:25.588+0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-20T16:48:26.849+0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_GOOGL> on 2025-10-20 13:45:40+00:00
[2025-10-20T16:48:26.923+0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'stock_etl', 'transform_GOOGL', 'manual__2025-10-20T13:45:40+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/stock_etl_dag.py', '--cfg-path', '/tmp/tmp50h7z0pr']
[2025-10-20T16:48:26.927+0300] {standard_task_runner.py:91} INFO - Job 45: Subtask transform_GOOGL
[2025-10-20T16:48:26.935+0300] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=223) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-20T16:48:26.936+0300] {standard_task_runner.py:64} INFO - Started process 233 to run task
[2025-10-20T16:48:35.815+0300] {task_command.py:426} INFO - Running <TaskInstance: stock_etl.transform_GOOGL manual__2025-10-20T13:45:40+00:00 [running]> on host 51be0a441cbc
[2025-10-20T16:49:00.522+0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='stock_etl' AIRFLOW_CTX_TASK_ID='transform_GOOGL' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T13:45:40+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-20T13:45:40+00:00'
[2025-10-20T16:49:00.524+0300] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T16:49:02.879+0300] {logging_mixin.py:188} WARNING - /opt/airflow/dags/stock_etl_dag.py:48 FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
[2025-10-20T16:49:04.749+0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T16:49:04.751+0300] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/stock_etl_dag.py", line 51, in transform_data
    if not results:
           ^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/generic.py", line 1519, in __nonzero__
    raise ValueError(
ValueError: The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().
[2025-10-20T16:49:06.807+0300] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=stock_etl, task_id=transform_GOOGL, run_id=manual__2025-10-20T13:45:40+00:00, execution_date=20251020T134540, start_date=20251020T134824, end_date=20251020T134906
[2025-10-20T16:49:08.297+0300] {standard_task_runner.py:110} ERROR - Failed to execute job 45 for task transform_GOOGL (The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().; 233)
[2025-10-20T16:49:09.126+0300] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2025-10-20T16:49:11.761+0300] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-20T16:49:12.015+0300] {local_task_job_runner.py:222} INFO - ::endgroup::
